requirements.txt

```diff
@@ -1,3 +1,4 @@
 dask
 numpy
 xarray
+zarr
```

Comment:
let's move this to `dev-requirements.txt` since it won't be required by xbatcher directly, just the tutorial.